src/sentry/static/sentry/less/group-detail.less

```diff
@@ -824,6 +824,12 @@
         box-shadow: inset 0 2px 0 rgba(0, 0, 0, 0.03);
         border-top: 0;
       }
+
+      .issue-time {
+        color: #968ba0;
+        margin-left: 5px;
+        font-size: 12px;
```

Comment:
We prefer to use styled components for new additions over adding into the shared less files. In `resolutionBox.jsx` you could add

```javascript
import styled from 'react-emotion';
import space from 'app/utils/space';
// ... rest of existing module

const StyledTimeSince = styled(TimeSince)`
  color: ${p => p.theme.gray2};
  margin-left: ${space(0.5)};
  font-size: ${p => p.theme.fontSizeSmall};
`;
```

Then use `<StyledTimeSince>` instead of `<TimeSince>`

Reply:
Thanks for the explanation, I've made this update. Cheers.